llvm/include/llvm/Support/TargetOpcodes.def

```diff
@@ -343,7 +343,7 @@ HANDLE_TARGET_OPCODE(G_FREEZE)
 HANDLE_TARGET_OPCODE(G_CONSTANT_FOLD_BARRIER)
 
 // INTRINSIC fptrunc_round intrinsic.
-HANDLE_TARGET_OPCODE(G_INTRINSIC_FPTRUNC_ROUND)
+HANDLE_TARGET_OPCODE(G_FPTRUNC_ROUND)
```

Comment:
Rename should be done separately. It is not necessary to move this to a pattern

Reply:
Agree! So don;t rename here in this patch. Done!